storage/Filesystems/MountableImpl.h

```diff
@@ -57,8 +58,16 @@ namespace storage
 	MountByType get_mount_by() const { return mount_by; }
 	void set_mount_by(MountByType mount_by);
 
-	const list<string>& get_fstab_options() const { return fstab_options; }
-	void set_fstab_options(const list<string>& fstab_options);
+	const MountOpts & get_mount_opts() const { return mount_opts; }
```

Comment:
Without swig parsing the header with MountOpts (and someday also CryptOpts) this will not work in the generated bindings. I propose to simply use ```vector<string>``` in the external API.